spec/site/tlc/alarm_spec.rb

```diff
@@ -40,6 +40,64 @@
       input_nr = Validator.config['alarm_activation'][alarm_code_id]  # what input to activate
       skip "alarm activation for alarm #{alarm_code_id}  not configured" unless input_nr
       
+      Validator::Site.connected do |task,supervisor,site|
+        prepare task, site
+        timeout  = Validator.config['timeouts']['alarm']
+
+        # first force input to false, so we're sure to get an alarm when we afterwards force to true
+        force_input_and_confirm input:input_nr, value:'False'
+
+        # the rsmp spec requires a specific casing of enums, but some equipment uses incorrect casing.
+        # incorrect casing is usualle cuaght by the json schema validation, but in case this is disabled,
+        # we use case-insensitive regex patterns so that the tests can still be run
+        mapping = {
+          'True' => /Active/i,    # when input is forced to True, the alarm should become active
+          'False' => /inActive/i  # when input is forced to False, the alarm should become inactive
+        }
+
+        mapping.each_pair do |input_value, alarm_status|
```

Comment:
i think the test can be simplified a bit, since we're not testing the activation? so instead of looping over input=>alarm pairs, just do:

- force input to false
- force input to true
- wait for an alarm
- acknowledge the alarm
- check that the alarm status becomes acknowledged
- release input